plugins/source/awspricing/plugin/plugin.go

```diff
@@ -0,0 +1,23 @@
+package plugin
+
+import (
+	"github.com/cloudquery/cloudquery/plugins/source/awspricing/client"
+	"github.com/cloudquery/cloudquery/plugins/source/awspricing/resources"
+	"github.com/cloudquery/plugin-sdk/plugins/source"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+var (
+	Version = "development"
+)
+
+func Plugin() *source.Plugin {
+	return source.NewPlugin(
+		"cloudquery-awspricing",
```

Comment:
```suggestion
		"awspricing",
```

I don't think we usually prefix the name